dayinfo.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+
+from errbot import botcmd, BotPlugin
+from datetime import datetime, date
+from calendar import isleap
+
+from pynameday.slovakia import Slovakia as sknameday
+from workalendar.europe import Slovakia as skholiday
```

Comment:
I feel like, in order to make this usable for more people, holidays (at least which calendar is to be used) should be configurable by users in per-channel config. What do you think @mrshu?